charts/secrets-store-csi-driver/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: secrets-store-csi-driver
-version: 0.0.12
-appVersion: 0.0.12
+version: 0.0.13
+appVersion: 0.0.13
```

Comment:
I don't think we need to update the `chartVersion` and `appVersion` here as long as we don't change any of the current defaults as part of making the images configurable